custom_md_loader/classes/MetadataMigrationController.cls

```diff
@@ -0,0 +1,222 @@
+    
+    private final Set<String> standardFieldsInHeader = new Set<String>();
+    private List<String> nonSortedApiNames;
+    
+    public boolean showRecordsTable{get;set;}
+    public String selectedType{get;set;}
+    public Blob csvFileBody{get;set;}
+    public SObject[] records{get;set;}
+    public List<SelectOption> cmdTypes{public get;set;}
+    public String selectedOpType1{get;set;}
+    public String selectedOpType2{get;set;}
+    public String selectedOpType3{get;set;}
+    public List<SelectOption> opTypes{public get;set;}
+    public List<SelectOption> objCreationOpTypes{public get;set;}
+    
+    public List<String> fieldNamesForDisplay{public get;set;}
+    
+    public String customSettingFromFieldAsIs{public get;set;}
+    public String customSettingFromFieldSimple{public get;set;}
+    public String cmdToFieldSimple{public get;set;}
+    public String customSettingFromFieldJson{public get;set;}
+    public String cmdToFieldJson{public get;set;}
+
+	public String csFieldObjCreation {public get;set;}
+	public String cmtFieldObjCreation {public get;set;}
+	public String opTypeFieldObjCreation {public get;set;}
+
+	public String csNameApexMetadata{public get;set;}
+    public String cmdNameApexMetadata{public get;set;}
+    public String jsonMappingApexMetadata{public get;set;}
+        
+    public String jsonMapping{public get;set;}
+    
+    public boolean asyncDeployInProgress{get;set;}
+
+    public boolean isMessage {get;set;}
+	
+	public MetadataOpType opType = MetadataOpType.APEXWRAPPER;
+	//public MetadataOpType opType = MetadataOpType.METADATAAPEX; 
+
+    public MetadataMigrationController() {
+    
+    	service.timeout_x = 40000;
+    
+		isMessage = false;
+    	asyncDeployInProgress = false;
+    	
+        showRecordsTable = false;
+        loadCustomMetadataMetadata();
+        
+        //No full name here since we don't want to allow that in the csv header. It is a generated field using type dev name and record dev name/label.
+        standardFieldsInHeader.add(AppConstants.DEV_NAME_ATTRIBUTE);
+        standardFieldsInHeader.add(AppConstants.LABEL_ATTRIBUTE);
+        standardFieldsInHeader.add(AppConstants.DESC_ATTRIBUTE);
+        
+        jsonMapping = '{'
+            + '        \"customsetting_field1__c\" : \"cmd_field1__c\",'
+            + '        \"customsetting_field2__c\" : \"cmd_field2__c\",'
+            + '        \"customsetting_field3__c\" : \"cmd_field3__c\"'
+            + '}';
+            
+        opTypes = new List<SelectOption>();
+		opTypes.add(new SelectOption(MetadataOpType.APEXWRAPPER.name(), 'Sync Operation'));
+		opTypes.add(new SelectOption(MetadataOpType.METADATAAPEX.name(), 'Async Operation'));
+		
+		objCreationOpTypes = new List<SelectOption>();
+		objCreationOpTypes.add(new SelectOption(MetadataOpType.APEXWRAPPER.name(), 'Sync Operation'));
+            
+    }
+    
+    public PageReference onLoad () {
```

Comment:
Do you still need this? I think it can be removed, along with the reference in the VF page.